src/app/components/article/MediaArticlesTeamArticles.js

```diff
@@ -19,14 +19,13 @@ const MediaArticlesTeamArticlesComponent = ({
   <>
     { textSearch && !articles.length ? (
       <div className={cx('typography-body1', styles.articlesSidebarNoArticle)}>
-        <DescriptionIcon style={{ fontSize: 'var(--font-size-h4)' }} />
-        <div>
-          <FormattedMessage
-            id="articlesSidebar.noResults"
-            defaultMessage="No results matched your search."
-            description="Message displayed on articles sidebar when search returns no articles."
-          />
-        </div>
+        <DescriptionIcon />
+        <FormattedMessage
+          tagName="div"
+          id="articlesSidebar.noResults"
```

Comment:
The namespace ID for this `<FormattedMessage />`'s should be `mediaArticlesTeamArticles.`

Reply:
done